README.md

```diff
@@ -102,5 +110,24 @@ docker run \
 docker logs -f vault-benchmark
 ```
 
-# Documentation
+## Documentation
+
 Documentation for `vault-benchmark` including usage and test configuration can be found in our [docs](docs/index.md)
+
+## Community Project Disclaimer
+
+### No Official Support
+
+Please note that we do not offer any official support for this project. If you encounter issues or have questions, we recommend reaching out to the community through discussion forums or other community channels. Community members are often helpful and responsive.
```

Comment:
The use of "we" is slightly vague. should we change that to Hashicorp? Maybe something more specific to: "Please note this is a community project. Hashicorp and Hashicorp Support do not offer any official support for the Vault-Benchmark project."

The second sentence "... encounter issues ..." are we not using the issues section of GitHub to field some of this?

Reply:
Good update.

Yeah ideally the issues tab will handle this.  I will link it directly (it is already linked lower in the notes)